### ISSUE TITLE
Similar Targets visualisation is broken in IE11

### ISSUE THREAD

**Issue by gkos-bio** [opentargets/webapp#237]
### Observed behaviour
Visualisation is stuck owing to an incompatible function call in IE11
E.g. NOD2 and IL4R
http://www.targetvalidation.org/target/ENSG00000167207

```
Object doesn't support property or method 'forEach'
```

```js
        .style('opacity', 1)
        .each('end', function (d) {
            var siblings = this.parentNode.childNodes;
            siblings.forEach(function (node) {
                if (node.nodeName === 'text') {
                    d3.select(node)
                        .transition()
                        .duration(1000)
                        .style('opacity', 1);
                }
            });
```

### Expected behaviour
The normal behaviour with all the bubbles.

### Environment
[How did you see the bug?]
* Version: production (3.4.4)
* Browser: IE11

**Comment by peatroot** [comment]
Likely cause is lack of IE support for `forEach` on `NodeList`s. See [this reference](https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach).

**Comment by peatroot** [comment]
Fixed in production (tag: `3.4.5`) and `master`.
